scripts/functional.js

```diff
@@ -79,16 +79,16 @@ const buildComponentFixtures = async () => {
   let browsers;
 
   if (chrome) {
-    browsers = "saucelabs:Chrome@latest:macOS 11.00";
+    browsers = "saucelabs:chrome@123:Mac 13";
```

Comment:
Seems the SauceLabs browser strings have changed. There is no longer a `@latest` tag. Also the OS strings changed.